src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -78,7 +78,7 @@ sortObjects(ExecutionSpace const &space, ViewType &view)
   if (result.min_val == result.max_val)
   {
     Kokkos::View<SizeType *, typename ViewType::device_type> permute(
-        Kokkos::ViewAllocateWithoutInitializing("permute"), n);
+        Kokkos::ViewAllocateWithoutInitializing("ArborX::Sorting::permute"), n);
```

Comment:
This does not follow the style of naming the label after the function it's in. Do we want to be flexible, or rigid by requiring it?

Reply:
If we decide to have a style document, we should also follow it. Otherwise, I am fine with some flexibility.

Reply:
Merging as is.  We can take this discussion to arborx/Wiki#14